src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/OrganisationServiceImpl.java

```diff
@@ -314,8 +313,27 @@ public OrganisationsDetailResponse findByOrganisationStatus(OrganisationStatus s
 
         }
         return new OrganisationsDetailResponse(organisations, true);
+
     }
 
+    @Override
+    @Transactional
+    public DeleteOrganisationResponse deleteOrganisation(Organisation organisation) {
+        DeleteOrganisationResponse deleteOrganisationResponse = null;
+        if (OrganisationStatus.PENDING.name().equalsIgnoreCase(organisation.getStatus().name())) {
```

Comment:
Both are enums and can be checked directly by == operator , don't need to use name method

Reply:
ok